### DISCUSSION TITLE
Dashed rectangle with other nodes nodes inside it.

### DISCUSSION THREAD

**Discussion by keithtobin** [Blazor-Diagrams/Blazor.Diagrams#303] [General]
It would be very welcome for any help.

I need to be able to create a dashed outline rectangle to represent a Azure network, this rectangle could be a network and have other nodes inside it. Also need to be able to create port nodes on outside of rectangle.

Like the follwing https://learn.microsoft.com/en-us/azure/architecture/reference-architectures/app-service-web-app/images/basic-webapp-v2.png

Thanks for any help or guidance

Also love the Diagram lib.

keith.

**Comment by zHaytam** [comment]
Hello,
Thank you for the kind words.

You will need to use the groups feature. Style your group however you want with CSS, in this case transparent background and dashed outline. Groups, just like nodes, can have ports and you can position them however you want so you can really do everything you need!

**Reply by keithtobin** [reply]
Thank you so much for the help, I have been playing with you library to get a better understanding. My next big hurdle is to figure how I can replace the default resize behavior of a Group, I know how to put 4 handles, one on each corner to resize. But I still have to figure how to override the default resize behavior.

**Reply by zHaytam** [reply]
Which version of the library are you using?

**Reply by keithtobin** [reply]
I am using the nugget 2.16, but if there is a newer package that make things easier, I can download and build source. Thanks.

**Reply by keithtobin** [reply]
Sorry that should be 2.1.6

**Reply by keithtobin** [reply]
I think I am begging to figure it out. I create new behaviors for custom resizing.

I am going to play and better understand behaviors.

And thanks for the lib, I like using it
